internal/ruleset/ruleset.go

```diff
@@ -0,0 +1,104 @@
+package ruleset
+
+import (
+	"embed"
+	"io/ioutil"
+	"log"
+	"path"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+//go:embed rulesets
+var ruleSetsFS embed.FS
+
+const rulesetsFolder = "rulesets"
+
+type RuleSet struct {
+	EmbeddedFiles []string
+	UserFiles     []string
+}
+
+func NewRuleSet(userFiles []string) (*RuleSet, error) {
+	var r = &RuleSet{}
+
+	embeddedFiles, err := loadEmbeddedFiles()
+	if err != nil {
+		return r, err
+	}
+
+	r.EmbeddedFiles = embeddedFiles
+	r.UserFiles = userFiles
+
+	return r, nil
+}
+
+func loadEmbeddedFiles() ([]string, error) {
```

Comment:
comment missing, and possibly misnamed. it isn't loading the files, its just enumerating (listing) them